.github/workflows/labelize.yml

```diff
@@ -0,0 +1,21 @@
+on: issue_comment
+name: Labelize
+jobs:
+  #----------------------------------------------------------------------------
+  # Labelize
+  #----------------------------------------------------------------------------
+  labelize:
+    name: Labelize
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request }}
+
+    steps:
+      - name: Comment on PR
+        if: contains(github.event.comment.body, '\label ci:')
+        shell: bash
+        run: |
+          label=$(cut -d " " -f2 <<< "${{ github.event.comment.body }}")
```

Comment:
```sh
> echo "hello arbitrary-label \label ci:blah" | cut -d " " -f2
arbitrary-label
```
![](https://c.tenor.com/zh58XZRJuzYAAAAd/tenor.gif)

Reply:
except you cant add a random label that doesnt exist.
A random user can add any label that start with `ci:` and already exist in the repo.

Reply:
> except you cant add a random label that doesnt exist.

indeed...

> A random user can add any label that start with `ci:` and already exist in the repo.

...but it looks like you can add _any_ random label that does exist (even if it's not `ci:*`)